### ISSUE TITLE
Allow log(c(x,y,z)) in initial estimates

### ISSUE THREAD

**Issue by mattfidler** [nlmixrdevelopment/nlmixr#253]
Hi @cokkies,  I'm going go go through things as I see them, so there may be many posts here.  You have:

```R
    tvS0 <- log(c(10, 21.8, 30))     #th(1)
```

While that should work I actually don't think it does currently.  Here is a test case:

``` r
library(nlmixr)

one.cmt <- function() {
  ini({
    ## You may label each parameter with a comment
    tka <- log(c(0,1.5,20)) # Log Ka
    tcl <- 1 # Log Cl
    ## This works with interactive models
    ## You may also label the preceding line with label("label text")
    tv <- 3.45; label("log V")
    ## the label("Label name") works with all models
    eta.ka ~ 0.6
    eta.cl ~ 0.3
    eta.v ~ 0.1
    add.sd <- 0.7
  })
  model({
    ka <- exp(tka + eta.ka)
    cl <- exp(tcl + eta.cl)
    v <- exp(tv + eta.v)
    linCmt() ~ add(add.sd)
  })
}

nlmixr(one.cmt)$ini
#> Error in nlmixrBounds(.ini): The following parameter names were duplicated: tka.
```

<sup>Created on 2019-11-14 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

However you can still do the same thing with taking the `log` of each parameter:

```` r
library(nlmixr)

one.cmt <- function() {
  ini({
    ## You may label each parameter with a comment
    tka <- c(log(0),log(1.5),log(20)) # Log Ka
    tcl <- 1 # Log Cl
    ## This works with interactive models
    ## You may also label the preceding line with label("label text")
    tv <- 3.45; label("log V")
    ## the label("Label name") works with all models
    eta.ka ~ 0.6
    eta.cl ~ 0.3
    eta.v ~ 0.1
    add.sd <- 0.7
  })
  model({
    ka <- exp(tka + eta.ka)
    cl <- exp(tcl + eta.cl)
    v <- exp(tv + eta.v)
    linCmt() ~ add(add.sd)
  })
}
nlmixr(one.cmt)$ini

#> Fixed Effects ($theta): 
#>     tka       tcl        tv 
#> 0.4054651 1.0000000 3.4500000 
#> 
#> Omega ($omega): 
#>       eta.ka eta.cl eta.v
#> eta.ka    0.6    0.0   0.0
#> eta.cl    0.0    0.3   0.0
#> eta.v     0.0    0.0   0.1

```

_Originally posted by @mattfidler in https://github.com/nlmixrdevelopment/nlmixr/issues/252#issuecomment-553938108_

**Comment by nsk64** [comment]
I think that the following line will give an error:
tka <- c(log(0),log(1.5),log(20)) # Log Ka
A more appropriate should be
tka <- c(log(0.0001),log(1.5),log(20)) # Log Ka

**Comment by billdenney** [comment]
This was fixed in #330.  It can be closed now.

**Comment by mattfidler** [comment]
Thanks @billdenney
